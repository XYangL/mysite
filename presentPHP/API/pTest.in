<h1>Download</h1>
	<p><a href="http://daringfireball.net/projects/downloads/Markdown_1.0.1.zip">Markdown 1.0.1</a> (18 KB) -- 17 Dec 2004</p>
	<p>A Cross Product Formula</p>
	<p>When \(a \ne 0\), there are two solutions to \[ax^2 + bx + c = 0\] and they are $$x = {-b \pm \sqrt{b^2-4ac} \over 2a}.$$</p>
<h1>Introduction</h1>
	<p>Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).</p>
	<p>Thus, "Markdown" is two things: (1) a plain text formatting syntax; and (2) a software tool, written in Perl, that converts the plain text formatting to HTML. See the <a href="/projects/markdown/syntax">Syntax</a> page for details pertaining to Markdown's formatting syntax. You can try it out, right now, using the online <a href="/projects/markdown/dingus">Dingus</a>.</p>
	<h2>Movable Type</h2>
		<p>Markdown works with Movable Type version 2.6 or later (including Movable Type 3.0).</p>
	<h2>Command-Line</h2>
		<p>Use the <code>--html4tags</code> command-line switch to produce HTML output from a Unix-style command line. E.g.:</p>
		<p>(You can use this '.text' suffix trick to view the Markdown source for the content of each of the pages in this section, e.g. the
		<a href="/projects/markdown/syntax.text">Syntax</a> and <a href="/projects/markdown/license.text">License</a> pages.)</p>
		<p>Markdown is free software, available under a BSD-style open source license. See the <a href="/projects/markdown/license">License</a> page for more information.</p>
<h1>Discussion List <a id="discussion-list"></a></h1>
	<p>I've set up a public <a href="http://six.pairlist.net/mailman/listinfo/markdown-discuss">mailing list for discussion about Markdown</a>. Any topic related to Markdown -- both its formatting syntax and its software -- is fair game for discussion. Anyone who is interested is welcome to join.</p>
	<p>It's my hope that the mailing list will lead to good ideas for future improvements to Markdown.</p>
<h1>Installation and Requirements <a id="install"></a></h1>
	<p>Markdown requires Perl 5.6.0 or later. Welcome to the 21st Century. Markdown also requires the standard Perl library module <a href="http://search.cpan.org/dist/Digest-MD5/MD5.pm">Digest::MD5</a>, which is probably already installed on your server.</p>
