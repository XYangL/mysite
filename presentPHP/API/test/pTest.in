<h1>Download</h1>
	<p><a href="http://daringfireball.net/projects/downloads/Markdown_1.0.1.zip">Markdown 1.0.1</a> (18 KB) -- 17 Dec 2004</p>
	<p>A Cross Product Formula</p>
	<p>When \(a \ne 0\), there are two solutions to \[ax^2 + bx + c = 0\] and they are $$x = {-b \pm \sqrt{b^2-4ac} \over 2a}.$$</p>
	<ol>
		<li><p>Rename the "Markdown.pl" plug-in to "Markdown" (case is important). Movable Type requires plug-ins to have a ".pl" extension; Blosxom forbids it.</p></li>
		<li><p>Copy the "Markdown" plug-in file to your Blosxom plug-ins folder. If you're not sure where your Blosxom plug-ins folder is, see the Blosxom documentation for information.</p></li>
		<li><p>That's it. The entries in your weblog will now automatically be processed by Markdown.</p></li>
		<li><p>If you'd like to apply Markdown formatting only to certain posts, rather than all of them, Markdown can optionally be used in conjunction with Blosxom's <a href="http://www.blosxom.com/plugins/meta/meta.htm">Meta</a> plug-in. First, install the Meta plug-in. Next, open the Markdown plug-in file in a text editor, and set the configuration variable <code>$g_blosxom_use_meta</code> to 1. Then, simply include a "<code>meta-markup: Markdown</code>" header line at the top of each post you compose using Markdown.</p></li>
	</ol>
<h1>Introduction</h1>
	<p>Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).</p>
	<p>Thus, "Markdown" is two things: (1) a plain text formatting syntax; and (2) a software tool, written in Perl, that converts the plain text formatting to HTML. See the <a href="/projects/markdown/syntax">Syntax</a> page for details pertaining to Markdown's formatting syntax. You can try it out, right now, using the online <a href="/projects/markdown/dingus">Dingus</a>.</p>
	<h2>Movable Type</h2>
		<p>Markdown works with Movable Type version 2.6 or later (including Movable Type 3.0).</p>
	<h2>Command-Line</h2>
		<p>Use the <code>--html4tags</code> command-line switch to produce HTML output from a Unix-style command line. E.g.:</p>
		<p>(You can use this '.text' suffix trick to view the Markdown source for the content of each of the pages in this section, e.g. the
		<a href="/projects/markdown/syntax.text">Syntax</a> and <a href="/projects/markdown/license.text">License</a> pages.)</p>
		<p>Markdown is free software, available under a BSD-style open source license. See the <a href="/projects/markdown/license">License</a> page for more information.</p>
<h1>Discussion List </h1>
	<p>I've set up a public <a href="http://six.pairlist.net/mailman/listinfo/markdown-discuss">mailing list for discussion about Markdown</a>. Any topic related to Markdown -- both its formatting syntax and its software -- is fair game for discussion. Anyone who is interested is welcome to join.</p>
	<p>It's my hope that the mailing list will lead to good ideas for future improvements to Markdown.</p>
<h1>Installation and Requirements</h1>
	<p>Markdown requires Perl 5.6.0 or later. Welcome to the 21st Century. Markdown also requires the standard Perl library module <a href="http://search.cpan.org/dist/Digest-MD5/MD5.pm">Digest::MD5</a>, which is probably already installed on your server.</p>
	<math >
	  <mi>a</mi><msup><mi>x</mi><mn>2</mn></msup>
	  <mo>+</mo> <mi>b</mi><mi>x</mi>
	  <mo>+</mo> <mi>c</mi> <mo>=</mo> <mn>0</mn>
	</math>
	<p>(You can use this '.text' suffix trick to view the Markdown source for the content of each of the pages in this section.</p>