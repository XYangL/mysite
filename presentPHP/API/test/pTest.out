<ul class="one_level">
	<li><a>Markdown Syntax</a></li>
	<li>
		<a>Download</a>
		<ul class="two_level">
			<li><a><span><bold href="http://daringfireball.net/projects/downloads/Markdown_1.0.1.zip">Markdown 1.0.1</bold> (18 KB) -- 17 Dec 2004</span></a></li>
			<li><a><span>A Cross Product Formula</span></a></li>
			<li><a><span>When \(a \ne 0\), there are two solutions to \[ax^2 + bx + c = 0\] and they are $$x = {-b \pm \sqrt{b^2-4ac} \over 2a}.$$</span></a></li>
		</ul>
	</li>
	<li>
		<a>Introduction</a>
		<ul class="two_level">
			<li><a><span>Markdown is a text-to-HTML conversion tool for web writers. Markdown allows you to write using an easy-to-read, easy-to-write plain text format, then convert it to structurally valid XHTML (or HTML).</span></a></li>
			<li><a><span>Thus, "Markdown" is two things: (1) a plain text formatting syntax; and (2) a software tool, written in Perl, that converts the plain text formatting to HTML. See the <bold href="/projects/markdown/syntax">Syntax</bold> page for details pertaining to Markdown's formatting syntax. You can try it out, right now, using the online <bold href="/projects/markdown/dingus">Dingus</bold>.</span></a></li>
			<li>
				<a>Movable Type</a>
				<ul class="three_level"><li><a><span>Markdown works with Movable Type version 2.6 or later (including Movable Type 3.0).</span></a></li></ul>
			</li>
			<li>
				<a>Command-Line</a>
				<ul class="three_level">
					<li><a><span>Use the <code>--html4tags</code> command-line switch to produce HTML output from a Unix-style command line. E.g.:</span></a></li>
					<li><a><span>(You can use this '.text' suffix trick to view the Markdown source for the content of each of the pages in this section, e.g. the
						<bold href="/projects/markdown/syntax.text">Syntax</bold> and <bold href="/projects/markdown/license.text">License</bold> pages.)
					</span></a></li>
					<li><a><span>Markdown is free software, available under a BSD-style open source license. See the <bold href="/projects/markdown/license">License</bold> page for more information.</span></a></li>
				</ul>
			</li>
		</ul>
	</li>
	<li>
		<a>Discussion List</a>
		<ul class="two_level">
			<li><a><span>I've set up a public <bold href="http://six.pairlist.net/mailman/listinfo/markdown-discuss">mailing list for discussion about Markdown</bold>. Any topic related to Markdown -- both its formatting syntax and its software -- is fair game for discussion. Anyone who is interested is welcome to join.</span></a></li>
			<li><a><span>It's my hope that the mailing list will lead to good ideas for future improvements to Markdown.</span></a></li>
		</ul>
	</li>
	<li>
		<a>Installation and Requirements</a>
		<ul class="two_level">
			<li><a><span>Markdown requires Perl 5.6.0 or later. Welcome to the 21st Century. Markdown also requires the standard Perl library module <bold href="http://search.cpan.org/dist/Digest-MD5/MD5.pm">Digest::MD5</bold>, which is probably already installed on your server.</span></a></li>
			<li><a><math><mi>a</mi><msup><mi>x</mi><mn>2</mn></msup><mo>+</mo><mi>b</mi><mi>x</mi><mo>+</mo><mi>c</mi><mo>=</mo><mn>0</mn></math></a></li>
			<li><a><span>(You can use this '.text' suffix trick to view the Markdown source for the content of each of the pages in this section.</span></a></li>
		</ul>
	</li>
</ul>
